src/components/ArticleActions.vue

```diff
@@ -8,6 +8,10 @@
     <button class="btn btn-outline-danger btn-sm" @click="deleteArticle">
       <i class="ion-trash-a"></i> <span>&nbsp;Delete Article</span>
     </button>
+    <span>&nbsp;&nbsp;</span>
+    <button class="btn btn-sm" @click="cloneArticle">
+      <i class="ion-copy-a"></i> <span>&nbsp;Clne Article</span>
```

Comment:
There is a typo here
```suggestion
      <i class="ion-copy-a"></i> <span>&nbsp;Clone Article</span>
```